src/manifest.json

```diff
@@ -33,7 +32,6 @@
         "expand": {
             "acting_user": "all",
             "channel": "summary",
-            "admin_access_token": "all",
```

Comment:
The token doesn't seem to be used: https://github.com/mattermost/mattermost-app-zendesk/blob/5c2d8805564e71480da21594e4b811c4d4f42cf0/src/bindings/index.ts#L18-L18